src/posix/platform/udp.cpp

```diff
@@ -323,6 +323,11 @@ otError otPlatUdpBindToNetif(otUdpSocket *aUdpSocket, otNetifIdentifier aNetifId
     case OT_NETIF_BACKBONE:
     {
 #if OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
+        if (otSysGetInfraNetifName() == 0 || otSysGetInfraNetifName() == 0)
```

Comment:
Seems to be checking same thing twice `if (otSysGetInfraNetifName() == 0 || otSysGetInfraNetifName() == 0)`?

Reply:
BTW, it is a pointer? Should we use `== nullptr` instead of `== 0`?

Reply:
+1 on both questions